pipeline.sh

```diff
@@ -6,12 +6,15 @@ function usage
 {
   echo "usage: pipeline.sh GFA_FILE [-b 00 -e 01 -s bSnSnS -w 1000 -t 12 -h]"
   echo "   ";
-  echo "  -b | --begin   : The start bin";
-  echo "  -e | --end     : The end bin";
-  echo "  -s | --sort    : Sort option on odgi";
-  echo "  -w | --width   : Bin width on odgi";
-  echo "  -t | --threads : Threads on odgi";
-  echo "  -h | --help    : This message";
+# echo "  -b | --begin          : The start bin";
+# echo "  -e | --end            : The end bin";
+  echo "  -s | --sort           : Sort option on odgi";
+  echo "  -w | --width          : Bin width on odgi";
+  echo "  -c | --cells-per-file : Cells per file on component_segmentation";
+  echo "  -t | --threads        : Threads on odgi";
+  echo "  -p | --port           : Pathindex port";
+  echo "  -h | --host           : Pathindex host";
+  echo "  -h | --help           : This message";
```

Comment:
Please change `-h | --host` to `-i | --host`

Reply:
I think this is the only open wrinkle here. The rest of the problems are not part of your code.
After fixing this, we can merge to master, I would suggest.

Reply:
Thanks, that's true.